sdv/metadata/multi_table.py

```diff
@@ -914,17 +914,20 @@ def _validate_foreign_keys(self, data):
 
         return [error_msg] if error_msg else []
 
-    def validate_data(self, data):
+    def validate_data(self, data, table_name=None):
```

Comment:
I think we shouldn't change the public `validate_data()` API by adding a parameter. Maybe we could add a private method that we can call instead.